trio_websocket/_impl.py

```diff
@@ -687,7 +687,8 @@ class WebSocketConnection(trio.abc.AsyncResource):
     def __init__(self, stream, ws_connection, *, host=None, path=None,
         client_subprotocols=None, client_extra_headers=None,
         message_queue_size=MESSAGE_QUEUE_SIZE,
-        max_message_size=MAX_MESSAGE_SIZE):
+        max_message_size=MAX_MESSAGE_SIZE,
+        receive_buffer_size=DEFAULT_RECEIVE_BYTES):
```

Comment:
It would need to be more than the `WebSocketConnection` constructor.  All API's that transitively call this in the library.  Search for `MAX_MESSAGE_SIZE` and follow those.

* connect_websocket()
  * connect_websocket_url()
* open_websocket()
    * open_websocket_url()
* wrap_client_stream()
* wrap_server_stream()
* WebSocketServer  (_handle_connection())
  * serve_websocket()